projects/snapshot_bot/memory.h

```diff
@@ -10,19 +10,31 @@
 #include "third_party/units.h"
 
 // Standard C++ includes
-#include <fstream>
+#include <string>
+#include <unordered_map>
+#include <vector>
 
 // Forward declarations
 class Config;
+class ImageInput;
 
 namespace BoBRobotics
 {
 namespace ImgProc
 {
 class Mask;
 }
+
+namespace Navigation
+{
+class ImageDatabase;
+}
 }
 
+// Bounds used for extracting masks from ODK2 images
+const cv::Scalar odk2MaskLowerBound(1, 1, 1);
+const cv::Scalar odk2MaskUpperBound(255, 255, 255);
```

Comment:
won't these be instantiated in every compilation unit and cause multiple definition linker errors?

Reply:
It does compile ok for me. But you're right that this is wrong! It might break only if they're not inlined...